src/app/item-page/alerts/item-alerts.component.spec.ts

```diff
@@ -84,4 +108,17 @@ describe('ItemAlertsComponent', () => {
       expect(privateWarning).toBeNull();
     });
   });
+
+  it('should return true when user is not an admin and there is at least one correction with topic REQUEST_REINSTATE', fakeAsync(() => {
+    const isAdmin = false;
+    const correction = [{ topic: 'REQUEST_REINSTATE' }];
+    authorizationService.isAuthorized.and.returnValue(of(isAdmin));
+    correctionTypeDataService.findByItem.and.returnValue(of(correction));
+
+    const result$ = component.showReinstateButton$();
+    tick();
+    result$.subscribe((result) => {
+      expect(result).toBeTrue();
+    });
```

Comment:
This test doesn't actually test anything. You can verify that's the case by changing `toBeTrue()` to `toBeFalse()`. It will still pass.

You'll likely need to use the [jasmine done function](https://angular.io/guide/testing-components-scenarios#jasmine-done), or a [testscheduler](https://rxjs.dev/guide/testing/marble-testing) to test this